hoot-test/src/main/cpp/hoot/test/ProcessPool.cpp

```diff
@@ -161,7 +161,7 @@ void ProcessThread::processJobs(JobQueue* queue)
           {
             _proc.reset(createProcess());
             _outMutex->lock();
-            cout << test.toStdString() << " failed, requeued." << endl;
+            cout << test.toStdString() << " failed to launch, requeued." << endl;
```

Comment:
Just better error explanation.  It didn't necessarily fail, it just failed to launch correctly.  That is why it is requeued.  Real failures aren't requeued, they are reported.